handlers/BES_Modules_The_HDF5_Handler.adoc

```diff
@@ -143,30 +478,393 @@ H5.IgnoreUnknownTypes is true. See above.
+/a/b                     Group
+/a/b/c                   Group
+
+----
+
+Since this file does not have variables so the DDS is empty. 
+The corresponding DAS is:
+----
+Attributes {
+    HDF5_ROOT_GROUP {
+        a {
+            b {
+                c {
+                }
+            }
+        }
+    }
+    /a/ {
+        String HDF5_OBJ_FULLPATH "/a/";
+    }
+    /a/b/ {
+        String HDF5_OBJ_FULLPATH "/a/b/";
+    }
+    /a/b/c/ {
+        String HDF5_OBJ_FULLPATH "/a/b/c/";
+    }
+}
+
+----
+The attribute container __HDF5_ROOT_GROUP__ preserves the information of the group hierarchy. 
+
+Another example show an HDF5 dataset with HDF5 compound datatype. The __h5dump__ header of the HDF5 file __d_compound.h5__ is:
+----
+HDF5 "d_compound.h5" {
+GROUP "/" {
+   DATASET "compound" {
+      DATATYPE  H5T_COMPOUND {
+         H5T_STD_I32BE "Serial number";
+         H5T_STRING {
+            STRSIZE H5T_VARIABLE;
+            STRPAD H5T_STR_NULLTERM;
+            CSET H5T_CSET_ASCII;
+            CTYPE H5T_C_S1;
+         } "Location";
+         H5T_IEEE_F64BE "Temperature (F)";
+         H5T_IEEE_F64BE "Pressure (inHg)";
+      }
+      DATASPACE  SIMPLE { ( 4 ) / ( 4 ) }
+      ATTRIBUTE "value" {
+         DATATYPE  H5T_COMPOUND {
+            H5T_STD_I32BE "Serial number";
+            H5T_STRING {
+               STRSIZE H5T_VARIABLE;
+               STRPAD H5T_STR_NULLTERM;
+               CSET H5T_CSET_ASCII;
+               CTYPE H5T_C_S1;
+            } "Location";
+            H5T_IEEE_F64BE "Temperature (F)";
+            H5T_IEEE_F64BE "Pressure (inHg)";
+         }
+         DATASPACE  SIMPLE { ( 4 ) / ( 4 ) }
+      }
+   }
+}
+----
+
+The corresponding DDS is:
+----
+Dataset {
+    Structure {
+        Int32 Serial%20number;
+        String Location;
+        Float64 Temperature%20%28F%29;
+        Float64 Pressure%20%28inHg%29;
+    } /compound[4];
+} d_compound.h5;
+----
+
+Note the HDF5 compound variable array __/compound__ maps to DAP's array of Structure. The special characters inside the member names of the compound datatype are changed according to section 5.1 <<Default Option: DAP4 Name>>.
+
+== BES Keys
+In the course of supporting easy access to NASA HDF5/HDF-EOS5/netCDF4 files via Hyrax, various performance and other optimization tuning options are provided to hyrax service customers via BES keys. In this section, the descriptions for critical BES keys are provided. For the comprehensive BES key description, check the HDF5 handler configuration file https://github.com/OPENDAP/bes/blob/master/modules/hdf5_handler/h5.conf.in[h5.conf.in] at github.
+
+=== Keys for Both Options
+
+H5.EnableCF:::
+ 
+  * When this key is set to __true__ or does not show up, The handler handle the HDF5 file by following the CF conventions. The handler is especially tuned to handle NASA HDF5/netCDF4/HDF-EOS5 data products. For the tested NASA products, see <<NASA Products supported and tested by the CF option of the Handler>>. 
```

Comment:
One thing I am noticing is that to get the "Default Behavior" for DAP2 is produced by `H5.EnableCF=false` However, if I am reading this correctly the default value is `H5.EnableCF=true` which makes me (and probably others) confused about what default means in these contexts. 

Also I think `does not show up` probably means `is not set in the configuration` and I think the language should be changed to the latter.